docker/build-tools/Dockerfile

```diff
@@ -191,16 +204,16 @@ RUN tar -xzf /tmp/node-v${NODEJS_VERSION}-headers.tar.gz --strip-components=1 -C
 
 RUN npm init -y
 RUN npm install --production --global \
-    sass@${SASS_VERSION} \
-    sass-lint@${SASS_LINT_VERSION} \
-    typescript@${TYPESCRIPT_VERSION} \
-    tslint@${TSLINT_VERSION} \
-    markdown-spellcheck@${MARKDONN_SPELLCHECK_VERSION} \
-    svgstore-cli@${SVGSTORE_CLI_VERSION} \
-    svgo@${SVGO_VERSION} \
-    @babel/core@${BABEL_CORE_VERSION} \
-    @babel/cli@${BABEL_CLI_VERSION} \
-    @babel/preset-env@${BABEL_PRESET_ENV_VERSION}
+    "sass@${SASS_VERSION}" \
```

Comment:
this would make more sense as `sass@"${SASS_VERSION}"`and all children.

Reply:
Done